python/MooseDocs/extensions/MooseCommonExtension.py

```diff
@@ -49,8 +49,8 @@ def getSettings(self, settings_line):
       SETTINGS_RE = re.compile("([^\s=]+)=(.*?)(?=(?:\s[^\s=]+=|$))")
       matches = SETTINGS_RE.findall(settings_line.strip())
 
-      if not matches:
-        return {}
+      if len(matches) == 0:
+          return self._settings, {}
```

Comment:
You need to return a copy of self._settings, otherwise the settings dict() can be modified between calls the the 'handleMatch' methods. For example, you match an image and call "pop" on something. This setting will no-longer be there for the next time an image is found.